arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceSelectWidget/components/ResourceInstanceSelectWidgetEditor.vue

```diff
@@ -123,17 +123,22 @@ function getOption(value: string): ResourceInstanceReference | undefined {
     return options.value.find((option) => option.resource_id == value);
 }
 
-function resolver(event: FormFieldResolverOptions) {
-    return getOption(event.value);
+function transformValue(event: FormFieldResolverOptions) {
+    const option = getOption(event.value);
+
+    return {
+        display_value: option ? option.display_value : "",
```

Comment:
We're using a localized "(Empty)" rather than "" in querysets right now, but this may have been a provenance specific requirement. @apeters, thoughts?

Reply:
@jacobtylerwalls I don't think Provenance "requires" the use of "Empty" as a notation of "null" values it's just something we chose.  I don't think we should encode anything like that here or in querysets though.

Reply:
Got it, so the frontend would always check for empty string and then replace it with whatever it wants/needs? I can update the API if so.

Reply:
https://github.com/archesproject/arches-querysets/issues/83